modules/openapi-generator/src/main/resources/php/ObjectSerializer.mustache

```diff
@@ -301,17 +325,26 @@ class ObjectSerializer
                 }
             }
             $instance = new $class();
-            foreach ($instance::openAPITypes() as $property => $type) {
-                $propertySetter = $instance::setters()[$property];
-
-                if (!isset($propertySetter) || !isset($data->{$instance::attributeMap()[$property]})) {
-                    continue;
+            $reverseAttributes = array_flip($instance::attributeMap());
+            if (!is_object($data)) {
+                return null;
+            }
+            if (!empty($instance::oneOf())) {
+                return self::deserialize($data, "oneOf[". implode(', ', $instance->onefOf())  ."]", $httpHeaders);
```

Comment:
convert a model consiting of a oneof block to plain type string. This of course does not deal with cases where the model would contain additional stuff.